File: pytools/data/data.code.java.client/ReqSceneReqPlayers.java
package protocol;


import network.Packet;
import network.PacketUtil;

import java.util.List;
import java.util.ArrayList;


public class ReqSceneReqPlayers
{


	public byte[] encode()
	{
		Packet packet = new Packet();
		return packet.encode((short)Msg.P_REQ_SCENE_REQ_PLAYERS);
	}



}

File: pytools/data/data.code.csharp.client/ReqSceneReqPlayers.cs
using System.Collections;
using System.Collections.Generic;


public class ReqSceneReqPlayers
{


	public Packet Encode()
	{
		Packet packet = new Packet();
		packet.Encode(Msg.P_REQ_SCENE_REQ_PLAYERS);
		return packet;
	}



}
